### PR TITLE
ExamFee entity model

### PR DESCRIPTION
**Pull request by bilalmehrban**
- ExamFee Model Setup & Migrations with EF Core
- ExamFee Broker Select By Id function
- ExamFee Broker Select All function 
- ExamFee Broker Insert function 
- ExamFee Broker Delete function 
- ExamFee Broker Update function

### CODE REVIEW

File: OtripleS.Web.Api/Migrations/20210413113346_ExamFee.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
+﻿using System;
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Migrations/20210413113346_ExamFee.cs:1]: Add the copyrights notes here please

File: OtripleS.Web.Api/Migrations/20210413121338_UpdatedExamFeeModel.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Migrations/20210413121338_UpdatedExamFeeModel.cs:1]: Add copyrights notice here as well

File: OtripleS.Web.Api/Models/ExamFees/ExamFee.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        public Guid CreatedBy {get; set;}
+        public User CreatedByUser {get; set;}
+        public Guid UpdatedBy {get; set;}
+        public User UpdatedByUser {get; set;}
+    }
+    public enum ExamFeeStatus
+    {
+        Active,
+        Inactive
+    }
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Models/ExamFees/ExamFee.cs:28]: Put this enum in it's own file under the same namespace/folder

**Comment by hassanhabib** [comment]
@bilalmehrban just a couple of comments.
@viralpandya please take a look at this PR as well when you get a moment.

**Review by viralpandya — APPROVED** [review]

File: OtripleS.Web.Api/Brokers/Storage/StorageBroker.ExamFees.References.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+using OtripleS.Web.Api.Models.ExamFees;
+
+namespace OtripleS.Web.Api.Brokers.Storage
+{
+    public partial class StorageBroker
+    {
+        private void AddExamFeeReferences(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<ExamFee>()
+                .HasOne(examFee => examFee.CreatedByUser)
```
Comment (viralpandya) [inline OtripleS.Web.Api/Brokers/Storage/StorageBroker.ExamFees.References.cs:16]: We need to create relation between (Exam, Fees) table with ExamFees

**Review by viralpandya — CHANGES_REQUESTED** [review]
Please add relationship between ExamFee <---> Fee and Exam <--> ExamFee.

**Pull request merged**